internal/store/store.go

```diff
@@ -316,8 +316,10 @@ func (s *Store) createV2Client(client *http.Client) error {
 		atlasv2.UseDebug(log.IsDebugLevel())}
 
 	if s.baseURL != "" {
-		opts = append(opts, atlasv2.UseBaseURL(s.baseURL))
+		urlWithoutSuffix := strings.TrimSuffix(s.baseURL, "/")
+		opts = append(opts, atlasv2.UseBaseURL(urlWithoutSuffix))
```

Comment:
can this be handled by the SDK, for example this is alreayd done by the manually generated one (tho in the inverse order or adding the missing slash)
https://github.com/mongodb/go-client-mongodb-atlas/blob/01d7604ca665b81f12e797bc8976ec1d76b0449d/mongodbatlas/mongodbatlas.go#L419-L421

Reply:
Sure

Reply:
https://github.com/mongodb/go-client-mongodb-atlas/pull/385/files

Reply:
Reverted current code.

Reply:
approved there